content/aips/CRV-INTEREST-RATES.md

```diff
@@ -0,0 +1,156 @@
+
+# Motivation
+
+With the CRV reserve on Polygon reaching its SupplyCap, this is an ideal time to implement an upgrade that addresses the SupplyCap and Interest Rate parameters. 
+
+The current CRV Interest Rate reflects the original parameters implemented when the liquidity pools were deployed. The Interest Rate Parameters do not take into consideration the alternative use cases for CRV and consequently, Aave has missed out on revenue from the sustained borrowing demand.
+
+Llama expects the sustained borrowing demand to continue due to Curve's veTokenomics presenting several opportunities for CRV borrowers to earn yield. 
+
+In this proposal, the Uoptimal parameter is to be increased from 45% to 70%, which improves the capital efficiency of active reserves.
+
+![](https://i.imgur.com/BhKxsuS.png)
+
+![](https://i.imgur.com/GZNc30u.png)
+
+Specific to the Polygon v3 deployment, increasing the Uoptimal parameter from 45% to 70% will lead to an increase in the BorrowCap, which is a function of the SupplyCap and Uoptimal parameter.
+
+BorrowCap = SupplyCap * (Uoptimal + 0.1)
+
+The new SupplyCap parameter has been prepared by Chaos Labs, 1,125.24K. This generates a new Borrow Cap of 900.19K, [2].
+
+![](https://i.imgur.com/VDtUak2.png)
+
+The interest rate at the current Optimal point is 7%.
+
+![](https://i.imgur.com/g0jtJql.png)
+
+This proposal recommends introducing a Base and increasing the Slope1 parameter to 3% and 14% respectively.  This will lead to a substantial change to the current curve. Over time, we will monitor the reserve and amend the Slope1 parameter to optimize for risk exposure and revenue generation.
+
+The graphic below shows the changes in the interest rate.
+
+![](https://i.imgur.com/WMT8KMn.png)
+
+The Reserve Factor on Polygon v3 is to be increased from 10% to 20%, bringing it in line with all other Aave deployments.
+
+# Specification
+
+The below table shows the current and proposed changes to the CRV Reserve on Polygon v3, Polygon v2 and Ethereum v2.
+
+**Polygon v3 - Active**
+
+|Parameter|Current|Proposed|
+| --- | --- | --- |
+|SupplyCap|937,700 units|1,125,240 units|
+|BorrowCap|640,440 units|900,190 units|
+|Uoptimal|45.00%|70.00%|
+|Variable Base|0.00%|3.00%|
+|Variable Slope1|7.00%|14.00%|
+|Variable Slope2|300.00%|300.00%|
+|Reserve Factor|10.00%|20.00%|
+|Stable Borrowing|Disabled|Disabled|
+|Stable Slope1|0.00%|8.00%|
+|Stable Slope2|0.00%|300.00%|
+|Base Stable Rate Offset|2.00%|3.00%|
+|Stable Rate Excess Offset|5.00%|5.00%|
+|Optimal Stable To Total Debt Ratio|20.00%|20.00%|
+
+**Polygon v2 - Frozen**
+
+|Parameter|Current |Proposed|
+| --- | --- | --- |
+|Uoptimal|45.00%|45.00%|
+|Base|0.00%|3.00%|
+|VariableSlope1|7.00%|14.00%|
+|VariableSlope2|300.00%|300.00%|
+|StableSlope1|10.00%|17.00%|
+|StableSlope2|300.00%|300.00%|
+|Reserve Factor|20.00%|20.00%|
+
+**Ethereum v2 - Frozen**
+
+|Parameter|Current (%)|Proposed (%)|
```

Comment:
it's a bit inconsistent with the other two headings ` (%)` would recommend to align one way or another.